sqlite_utils/db.py

```diff
@@ -729,7 +729,7 @@ def enable_fts(self, columns, fts_version="FTS5"):
     def populate_fts(self, columns):
         sql = """
             INSERT INTO "{table}_fts" (rowid, {columns})
-                SELECT rowid, {columns} FROM {table};
+                SELECT rowid, {columns} FROM "{table}";
         """.format(
```

Comment:
I've been using [...] escaping instead of "..." escaping elsewhere, so let's do that here as well for consistency.